tests/end2end/playwright/dnd-form.spec.js

```diff
@@ -1,89 +1,143 @@
 // @ts-check
 import { test, expect } from '@playwright/test';
+import {ProjectPage} from "./pages/project";
+import {editedFeatureIds} from "./globals";
 
-test.describe('Drag and drop from', function () {
-    test.beforeEach(async function ({ page }) {
-        // Runs before each tests in the block
-        await page.goto(
-            '/index.php/view/map/?repository=testsrepository&project=dnd_form'
-        );
-        await page.locator('#button-attributeLayers').click();
+
+test.describe(
+    'Drag and drop from, editing test data',
+    {
+        tag: ['@write'],
+    }, () =>
+{
+
+    test('With new geom data creation, not remove data', async function ({ page }) {
+        const project = new ProjectPage(page, 'dnd_form');
+        await project.open();
+        await project.openEditingFormWithLayer('dnd_form_geom');
+        await project.dock.getByText('tab2').click();
+        await project.editingField('field_in_dnd_form').fill('value in DND form');
+        await project.clickOnMapLegacy(600, 200);
+
+        await project.editingSubmitForm();
+        const ids = await editedFeatureIds(page);
+
+        await project.clickOnMap(600, 200);
+        await expect(project.popupContent).toBeVisible();
+
+        // Fixme if there is a better way to get the table from the ID in the cell
+        // if the test is run multiple times, Postgres will increment the ID
+        const table = project.popupContent.getByRole("row", { name: ids['id'] }).locator("..");
```

Comment:
```suggestion
        const table = await project.popupContent.getByRole("row", { name: ids['id'] }).locator("..");
```

`getByRole` in an `async` method.